src/bundle/Resources/views/themes/admin/ui/field_type/preview/content_fields.html.twig

```diff
@@ -447,7 +447,7 @@
                         <td>{{ 'ezimage.master_dimensions'|trans|desc('Master dimensions') }}:</td>
                         <td>{{ 'ezimage.width_and_height'|trans({'%width%': field.value.width, '%height%': field.value.height})|desc('Width: %width%px height: %height%px') }}</td>
                     </tr>
-                    {% if field.value.height != '0' %}
+                    {% if field.value.height != '0' and field.value.height != '' %}
```

Comment:
This should achieve the same result:
```suggestion
                    {% if field.value.height %}
```

However, the important bit is that what is in the block below relies on `field.value.height` being "not zero", as we're performing a division. Non-empty string that does not contain a numerical value will cause PHP to error out.

As Fabien pointed out in the comments to the PR you linked, adding `is_numeric` as a function/test to Twig is trivial and we can do that relatively easily.

Reply:
Yes, we need to keep the check for value `0` in order to avoid division by zero.
But we could change it to this indeed:

```suggestion
                    {% if field.value.height != '0' and field.value.height %}
```

Or do you prefer I make a `is numeric` test to Twig so it reads something like :

```suggestion
                    {% if field.value.height is numeric and field.value.height != '0' %}
```

Reply:
@Steveb-p You want me to make a `is numeric` test, or this is not needed ?

Reply:
I'd stay away from adding a `numeric` test, since we would be introducing it to every installation. And we can't guarantee someone else did not have it implemented already.

I'd fine with the first set you shown, although I'm pretty sure `"0"` is a falsy value and that means 
```twig
{% if field.value.height %}
```
is logically equal to
```twig
{% if field.value.height != '0' and field.value.height %}
```

Reply:
You are right. They are logical the same.